mautic/mautic
# Stage - Add weight and contacts column

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
PR added a weight column and a clickable number of contacts column for the stage.

![image](https://github.com/mautic/mautic/assets/462477/a1971e95-69a2-4353-8c09-a3d59b475daf)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open stages, see new changes
3. Try add  and remove some contacts from stages
4. Check If counts are correct
